convert.go

```diff
@@ -0,0 +1,140 @@
+package godash
+
+import (
+	"strconv"
+)
+
+type AnyType interface {
+	convert()
+}
+
+// *******************************************************
+// ************** Convert string to integer **************
+// *******************************************************
+
+func ToInt(str string) (int, error) {
+	// Covert a string to a base-10 integer
+	val, err := strconv.ParseInt(str, 0, 0)
+	if err != nil {
+		// This error handling also takes care of overflows. If string is out of range for the int type,
+		// strconv will throw an "ErrRange" error, indicating the overflow.
+		return 0, INT_ERR
```

Comment:
What's the reasoning for making our own errors for these? Why not return the error that strconv returns?

Reply:
This was mostly because `strconv` only has two `error`s, a generic and a out-of-range error. However, both errors concatenate the calling functions's name (if I remember correctly), and so for testing, it wasn't enough to rely on `strconv`'s errors. So I made a more generic and standard error.

Reply:
For example:` i, e := strconv.ParseInt("1000000", 10, 8)` results in `e.Error()` being equal to `strconv.ParseInt: parsing "1000000": value out of range`